index.html

```diff
@@ -73,8 +73,246 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
             </section>
         </div>
 
+    </div>
+    <div class="cards grid-container">
+        <section class="first-card card">
+            <img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style">
+                    Tonic
+                </h1>
+                <section class="card-description">
+                    <ul class="font-style">
+                        <li class="card-list-title"> CANOPY</li>
+                        <li class="Job-title_and_year">
+                            Back End Dev
+                        </li>
+                        <li class="Job-title_and_year">
+                            2022
+                        </li>
+                    </ul>
+                </section>
+                <p class="notice font-style">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programming-languages font-style">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project-button" type="button">See Project</button>
+            </section>
```

Comment:
- [ ] Great job here 💯 However, according to the requirements and your Figma design you should Implement the button interactions (enable, hover, etc.). So kindly add them 👍

![image](https://user-images.githubusercontent.com/45231276/200123610-1de85b20-5012-428a-96f7-7f81931ec3bd.png)

- [ ] Kindly, don't forget to apply it to all your buttons 👍